CONTROLLERS/articles.controllers.js

```diff
@@ -1,14 +1,26 @@
 /* eslint-disable prefer-promise-reject-errors */
-const {fetchArticleById} = require('../MODELS/articles.models');
+const {fetchArticleById, updateVotesByArticleID} = require('../MODELS/articles.models');
 
 exports.getArticleByID = (req, res, next) => {
   const {article_id} = req.params;
   return fetchArticleById(article_id)
       .then((article)=>{
         return res.status(200).send({article});
       })
-      .catch((err) =>{
-        next(err);
-      });
+      .catch(next);
 };
 
+exports.patchVotesByArticleID = (req, res, next) => {
+  const {inc_votes} = req.body;
+  const {article_id} = req.params;
+
+  return updateVotesByArticleID(inc_votes, article_id)
+      .then((updatedArticle)=> {
+        return res.status(201).send({updatedArticle});
```

Comment:
this key should match the key used in fetchByArticleId or you may see some issues in the future!